### PR TITLE
Add numpy bincount docstrings

### PR DESCRIPTION
**Pull request by dranaivo**
Related to #481.

Added a docstring to `keras_core.ops.bincount` / `keras_core.ops.numpy.bincount`.

### CODE REVIEW

File: keras_core/ops/numpy.py [hunk 1]
```diff
@@ -1197,6 +1197,47 @@ def compute_output_spec(self, x):
...
+            `max(x)` is set to 0.
+
+    Returns:
+        1D tensor where each element gives the number of occurrence(s) of its
+        index value in x. Its length is the maximum between `max(x) + 1` and 
+        minlength.
+
+    Examples:
+        >>> x = []
+        >>> for i in range(4): x.extend(x.extend([i]*i))
```
Comment (fchollet) [inline keras_core/ops/numpy.py:1223]: This is pretty hard to read -- not a great fit for a docstring example

File: keras_core/ops/numpy.py [hunk 2]
```diff
@@ -1197,6 +1197,47 @@ def compute_output_spec(self, x):
...
+            `max(x) + 1`, each value of the output at an index higher than
+            `max(x)` is set to 0.
+
+    Returns:
+        1D tensor where each element gives the number of occurrence(s) of its
+        index value in x. Its length is the maximum between `max(x) + 1` and 
+        minlength.
+
+    Examples:
+        >>> x = []
```
Comment (fchollet) [inline keras_core/ops/numpy.py:1222]: Example code should be at the same level of indent as the docstring

File: keras_core/ops/numpy.py [hunk 3]
```diff
@@ -1197,6 +1197,47 @@ def compute_output_spec(self, x):
...
+        >>> for i in range(4): x.extend(x.extend([i]*i))
+        >>> x = keras_core.ops.array(x)
+        >>> x.numpy()
```
Comment (fchollet) [inline keras_core/ops/numpy.py:1225]: `.numpy()` is TF syntax, but code examples should be backend agnostic. You can just remove this line

File: keras_core/ops/numpy.py [hunk 4]
```diff
@@ -1197,6 +1197,47 @@ def compute_output_spec(self, x):
...
+        array([1, 2, 2, 3, 3, 3, 4, 4, 4, 4])
+        >>> keras_core.ops.bincount(x_arr).numpy()
+        array([0, 1, 2, 3, 4], dtype=int32)
+        >>> keras_core.ops.bincount(keras_core.ops.array([-1]))
+        [...] INVALID_ARGUMENT: Input arr must be non-negative!
```
Comment (fchollet) [inline keras_core/ops/numpy.py:1230]: No need to demonstrate invalid use cases, the error messages users get are self explanatory

File: keras_core/ops/numpy.py [hunk 5]
```diff
@@ -1197,6 +1197,47 @@ def compute_output_spec(self, x):
...
+    Args:
+        x: Input tensor.
+            It must be of dimension 1, and it must only contain non-negative
+            integer(s).
+        weights: Weight tensor.
+            It must have the same length as `x`. The default value is `None`.
+            If specified, `x` is weighted by it, i.e. if `n = x[i]`,
+            `out[n] += weight[i]` instead of the default behavior `out[n] += 1`.
+        minlength: An integer.
+            The default value is `0`. If specified, there will be at least
```
Comment (fchollet) [inline keras_core/ops/numpy.py:1211]: No need to use backticks for 0 (which is also not done for the value 0 on the last line here)

**Review by fchollet — COMMENTED** [review]
Thank you for the PR!

File: keras_core/ops/numpy.py (hunk 1 above)
Comment (dranaivo) [reply keras_core/ops/numpy.py:1223]: Is it alright like this :
```python
>>> x = keras_core.ops.array([1, 2, 2, 3, 3, 3, 4, 4, 4, 4])
```
?

**Review by fchollet — APPROVED** [review]
LGTM, thank you!

**Pull request merged**